mypy/modulefinder.py

```diff
@@ -350,6 +351,11 @@ def _find_module_non_stub_helper(
             if not self.fscache.isdir(dir_path):
                 break
         if plausible_match:
+            if self.options:
+                module_specific_options = self.options.clone_for_module(id)
+                if module_specific_options.follow_untyped_imports:
+                    # print("Returning ", id)
```

Comment:
No need for this comment

Reply:
Thanks for pointing this out, overlooked it when I removed my debugging statements.